### PR TITLE
fix indentation

### PR DESCRIPTION
**Pull request by chiarazampolli**

### CODE REVIEW

File: prodtests/full-system-test/aggregator-workflow.sh [hunk 1]
```diff
@@ -101,21 +101,21 @@ if workflow_has_parameter CALIB_PROXIES; then
...
+          echo "ERROR: You cannot run the TPC IDCs in FLP mode if you are not in EPNSYNCMODE" 1>&2
+          exit 2
+        fi
+        # define port for FLP; should be in 47900 - 47999; if nobody defined it, we use 47900
+        [[ -z $TPC_IDC_FLP_PORT ]] && TPC_IDC_FLP_PORT=47900
+        # expand FLPs; TPC uses from 001 to 145, but 145 is reserved for SAC
+        for flp in $(seq -f "%03g" 1 144)
+	do
+            FLP_ADDRESS="tcp://alicr1-flp-ib${flp}:${TPC_IDC_FLP_PORT}"
+            CHANNELS_LIST+=" --channel-config \"type=pull,name=tpcidc_flp${flp},transport=zmq,address=$FLP_ADDRESS,method=connect,rateLogging=10\""
```
Comment (davidrohr) [inline prodtests/full-system-test/aggregator-workflow.sh:114]: @chiarazampolli : This is indentend by 4 spaces, not by 2 like the test.
And I think part of the indentation you put is with tabs, and not with spaces. Could you perhaps check what your editor is doing? Perhaps it is configured to use tabs in some cases?

**Comment by ktf** [comment]
Please do not use tabs, use spaces.

File: prodtests/full-system-test/aggregator-workflow.sh (hunk 1 above)
Comment (chiarazampolli) [reply prodtests/full-system-test/aggregator-workflow.sh:114]: > Please do not use tabs, use spaces.

I don't do anything, the editor is indenting automatically.

Comment (chiarazampolli) [reply prodtests/full-system-test/aggregator-workflow.sh:114]: > @chiarazampolli : This is indentend by 4 spaces, not by 2 like the test. And I think part of the indentation you put is with tabs, and not with spaces. Could you perhaps check what your editor is doing? Perhaps it is configured to use tabs in some cases?

This was not the problem, I had one line still not correct after my fix. 
Out of curiosity, why tabs are not ok? When emacs indents automatically, all seems fine. then I commit, and it is all messed up.

Comment (chiarazampolli) [reply prodtests/full-system-test/aggregator-workflow.sh:114]: Now the command suggested by the CI moved the `do` in a funny position:

```
zampolli@pcalicecz5:~/SOFT/alibuild/ali-o2-dev1/O2$ sed -i 's/\t/  /g'  prodtests/full-system-test/aggregator-workflow.sh
zampolli@pcalicecz5:~/SOFT/alibuild/ali-o2-dev1/O2$ git diff
diff --git a/prodtests/full-system-test/aggregator-workflow.sh b/prodtests/full-system-test/aggregator-workflow.sh
index 70f486f28c..91a160495f 100755
--- a/prodtests/full-system-test/aggregator-workflow.sh
+++ b/prodtests/full-system-test/aggregator-workflow.sh
@@ -109,7 +109,7 @@ if workflow_has_parameter CALIB_PROXIES; then
         [[ -z $TPC_IDC_FLP_PORT ]] && TPC_IDC_FLP_PORT=47900
         # expand FLPs; TPC uses from 001 to 145, but 145 is reserved for SAC
         for flp in $(seq -f "%03g" 1 144)
-       do
+  do
             FLP_ADDRESS="tcp://alicr1-flp-ib${flp}:${TPC_IDC_FLP_PORT}"
             CHANNELS_LIST+=" --channel-config \"type=pull,name=tpcidc_flp${flp},transport=zmq,address=$FLP_ADDRESS,method=connect,rateLogging=10\""
         done
```
and the test is passed. I can leave it as it is, but I find it wrong.

Comment (chiarazampolli) [reply prodtests/full-system-test/aggregator-workflow.sh:114]: I don't know why the `do` was moved, to me it looks wrong, but the tests are happy with it. Can we merge this one, or do you see syntax errors? The code works:
```
for flp in $(seq -f "%03g" 1 3)
do
echo "FLP = $flp"
done
```
produces:
```
FLP = 001
FLP = 002
FLP = 003
```

**Comment by chiarazampolli** [comment]
Closing, @martenole fixed it here https://github.com/AliceO2Group/AliceO2/pull/9831

**Pull request closed without merging**